migrator/database/DatabaseImporter.java

```diff
@@ -116,15 +128,28 @@ public void run() {
             if (!checksum.verify(status)) throw TypeDBException.of(IMPORT_CHECKSUM_MISMATCH, checksum.mismatch(status));
             Instant end = Instant.now();
             LOG.info("Finished in: " + Duration.between(start, end).getSeconds() + " seconds");
-            LOG.info("Imported: " + status.toString());
+            LOG.info("Imported: " + status);
```

Comment:
Would it be useful to give the database name, in parallel to giving the filenames on export?

Reply:
Can add